packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/DefaultTransaction.tsx

```diff
@@ -80,7 +80,7 @@ function ShowGenericConfirmationScreen({ disabled, handleExampleSubmitted }: Req
   }, [params, handleExampleSubmitted]);
 
   return (
-    <Accordion.Item eventKey="5">
+    <Accordion.Item eventKey="1">
```

Comment:
just updated these keys so they are in the order they currently appear in accordion